dist_tables/dml.rst

```diff
@@ -38,6 +38,70 @@ For example:
 
     If COPY fails to open a connection for a shard placement then it behaves in the same way as INSERT, namely to mark the placement(s) as inactive unless there are no more active placements. If any other failure occurs after connecting, the transaction is rolled back and thus no metadata changes are made.
 
+INSERT INTO ... SELECT
```

Comment:
@begriffs and I chatted about this and I'm summarizing our chat below.

(1) We're communicating three concepts in this section: (a) what are roll ups, (b) how do they apply in a distributed setting with table colocation, and (c) how can you do them incrementally? Could we talk about them in this order? 

I usually find that (c) is usually more advanced than other concepts. That's one of the reasons we didn't mention them in our [reference architecture](https://docs.citusdata.com/en/v6.0/tech_soln/real_time_dashboards.html)

(2) Our current ``INSERT INTO ... SELECT`` implementation assumes that the source and destination tables are colocated. Could we mention this concept earlier on?

(3) Could we standardize on a diagram that demonstrates the concept of colocation? (We could optionally add a diagram for incremental upserts after this step.)

(4) When I think about this section, I'm slightly inclined to roll-up on ``tenant_id``. This is because our data modeling section talks about ``tenant_id`` vs ``entity_id``. When we mention ``page_id``, it feels a bit low in the hierarchical data model to me.